tests/SprykerEcoTest/Zed/ComputopApi/_support/_generated/ComputopApiZedTesterActions.php

```diff
@@ -1,4 +1,4 @@
-<?php  //[STAMP] 7eae624a25251dc8063b13f433655e7e
```

Comment:
Maybe we do not need generated file in repo?

Reply:
It is already on master. Not sure if I should delete it.